core/rpc-servers/src/lib.rs

```diff
@@ -30,9 +30,12 @@ const MAX_PAYLOAD: usize = 15 * 1024 * 1024;
 /// Default maximum number of connections for WS RPC servers.
 const WS_MAX_CONNECTIONS: usize = 100;
 
-type Metadata = apis::metadata::Metadata;
-type RpcHandler = pubsub::PubSubHandler<Metadata>;
+pub type Metadata = apis::metadata::Metadata;
+pub type RpcHandler = pubsub::PubSubHandler<Metadata>;
+
+#[cfg(not(target_os = "unknown"))]
```

Comment:
Wouldn't it be better to have a module with only single `#[cfg]` and reexport everything from that module, i.e.:
```rust
#[cfg(not(target_os = "unknown"))]
mod inner {
  pub type Metadata = ...;
  ...
  
}

#[cfg(target_os = "unknown")]
mod inner {}

pub use self::inner::*;